bin/matrix-list.js

```diff
@@ -30,12 +30,20 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
       });
     } else if (target.match(/app/)) {
       Matrix.validate.device(); //Make sure the user has selected a device
-      Matrix.firebase.app.list(function (err, data) {
+      Matrix.firebase.app.list(function (err, apps) {
         Matrix.loader.stop();
         if (err) return console.error('- ', t('matrix.list.app_list_error') + ':', err);
-        if (_.isUndefined(data)) data = {};
-        console.log(Matrix.helpers.displayApps(data));
-        process.exit();
+        if (_.isUndefined(apps)) apps = {};
+        async.forEach(Object.keys(apps), function (appId, done) { 
+          Matrix.firebase.app.getStatus(appId, function (status) { 
+            console.log("STATUS WATCH TRIGGERED! " + Matrix.config.user.id + '>' + Matrix.config.device.identifier + '>' + appId + '>' + status);
```

Comment:
Description updated, added inline comments, cleaned a bit, not sure how that console.log slipped through 😁